lib/sorbet-rails/routes_rbi_formatter.rb

```diff
@@ -15,20 +15,21 @@ def section(routes)
   def header(routes)
   end
 
-  def no_routes(routes, filter)
+  def no_routes(routes, filter=nil)
+    puts routes
```

Comment:
debug cruft or intentional?

Reply:
It's intentional, 5.1.7 only pass the routes and not `filter` to this method.